brvs-core/src/main/java/iroha/validation/rest/RestService.java

```diff
@@ -98,41 +104,35 @@ public Response getStatus(@PathParam("txHash") String hash) {
     if (transactionVerdict == null) {
       transactionVerdict = ValidationResult.UNKNOWN;
     }
-    return Response.status(HttpStatus.SC_OK).entity(transactionVerdict).build();
+    return Response.ok(new ValidationResultResponse(transactionVerdict)).build();
   }
 
   @POST
   @Path("/register/{accountId}")
+  @Produces(MediaType.APPLICATION_JSON)
   public Response register(@PathParam("accountId") String accountId) {
     try {
       registrationProvider.register(accountId);
-    } catch (Exception e) {
-      return Response.status(HttpStatus.SC_UNPROCESSABLE_ENTITY).entity(e).build();
+    } catch (InterruptedException e) {
+      throw new RuntimeException(e);
```

Comment:
I think we should return instead BrvsException with code REGISTRATION_FAILED

Reply:
Okay